lib/logstash-logger/formatter/base.rb

```diff
@@ -25,7 +25,9 @@ def build_event(message, severity, time)
                   when LogStash::Event
                     data.clone
                   when Hash
-                    event_data = data.merge("@timestamp".freeze => time)
+                    event_data = data.clone
+                    event_data['message'.freeze] = event_data.delete(:message) if event_data.key?(:message)
```

Comment:
My recollection is that Ruby will optimize this sort of code anyway, and therefore extracting to a constant isn't necessary. Can't find a source for this.